ntbs-service/ntbs-service.csproj

```diff
@@ -5,6 +5,7 @@
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
     <RootNamespace>ntbs_service</RootNamespace>
     <UserSecretsId>3736039d-9d74-48f5-9007-015e53f0b2a1</UserSecretsId>
+    <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
```

Comment:
I've found that sometimes Visual Studio has been refusing to build the project because of the typescript files. (Has anyone else noticed this? It's really weird that it's only happening some of the time!) Setting this flag removes the error.

Reply:
I've never seen it not build because of this, but glad we can suppress it!